src/pages/individual/SearchIndividuals.jsx

```diff
@@ -19,6 +19,8 @@ export default function SearchIndividuals() {
     reverse: true,
   });
 
+  console.log('SearchIndividuals.jsx: formFilters: ', formFilters);
```

Comment:
_might_ be left-behind debugging. but i am fine with leaving this if there is some on-going need to keep an eye on this.

Reply:
thank you. removed.